core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -70,7 +71,8 @@
                                summary="defaultOutcome|summarizeDefaultOutcome:getCurrentInteractionId():answerGroups.length:false"
                                short-summary="defaultOutcome|summarizeDefaultOutcome:getCurrentInteractionId():answerGroups.length:true"
                                labelled-as-correct="false"
-                               outcome="defaultOutcome">
+                               outcome="defaultOutcome"
```

Comment:
Btw is defaultOutcome always a cross?

Reply:
Yep! Since it's not actually stored in the answerGroups, its always considered to be incorrect. Can we not always categorize the correct option on some grounds and make an answerGroup from that ?

Reply:
I think we should actually allow the defaultOutcome to be correct (e.g. "pick a fraction that's not equivalent to X"). I'll see if I can do a migration PR for this.

Reply:
Yep! In that case it should be correct.